### PR TITLE
New lewd reagent: Xeno fluid

### PR DESCRIPTION
**Pull request by Kabramen**
## About The Pull Request

Adds a new selectable reagent for your bits, made in mind to be used for xeno characters: xeno fluid. 

image
image
 
And it **glows** in the dark!

image

*I was going to make another reagent but i got lazy.*

## Why It's Good For The Game

RP and stuff.

## Changelog
:cl:
add: xeno fluid.
/:cl:

### CODE REVIEW

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/vagina, GLOB.vagina_shapes_list)
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/breasts, GLOB.breasts_shapes_list)
 	GLOB.breasts_size_list = list ("a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o") //We need the list to choose from initialized, but it's no longer a sprite_accessory thing.
-	GLOB.genital_fluids_list = list ("Milk", "Water", "Semen", "Femcum", "Honey")
+	GLOB.genital_fluids_list = list ("Milk", "Water", "Semen", "Femcum", "Honey", "Xeno fluid")
```
Comment (DragonTrance) [inline code/__HELPERS/global_lists.dm:58]: "if you make glowing cum i'm pulling the plug on the server"
- quotefox

Comment (Kabramen) [reply code/__HELPERS/global_lists.dm:58]: image
image

Comment (quotefox) [reply code/__HELPERS/global_lists.dm:58]: its cum, you lose.

**Comment by quotefox** [comment]
No green cum

**Pull request closed without merging**